### DISCUSSION TITLE
Chaining selectors

### DISCUSSION THREAD

**Discussion by DMazona** [webdriverio/codemod#36] [Migration Issue]
I'm evaluating migration from protractor. I have several page objects in protractor where i define a common element and then use that element to define other elements. 
after conversion, it looks like this
```
    radiobutton1 = $('div[data-fieldid="RADIOBUTTON1"][class*=radiobutton]')
    radiobutton1_selected = this.radiobutton1.$("span=X")
```
vscode complains about this. though the webdriver documentation mentions being able to chain dollar selectors. Is there an easy way around this, other than redefining all of my selectors?

**Comment by christian-bromann** [comment]
> vscode complains about this.

Can you provide more information about the error VSCode is throwing? The example looks correct to me.

**Reply by DMazona** [reply]
Property '$' does not exist on type 'Promise<Element>'.ts(2339)

It suggest using await, but I can't use await, my page objects are properties
